DOCUMENTATION.md

```diff
@@ -0,0 +1,16 @@
+# Kepf Family Grocery API
+
+This API is a CRUD to manage a list of games.
+
+## API Endpoints
+
+* The following is a list of all the endpoints contained within the API
+
+### [GET][post] [PUT][delete]
+
+| URL                   |       DESCRIPTION        |   DATA    |
+| :-------------------- | :----------------------: | :-------: |
+| /api/game/create      |      Create a game       | models.js |
+| /api/game/get         | Retrieve a list of games | models.js |
+| /api/game/update      |       edit a game        | models.js |
+| /api/game/destroy/:id |      delete a game       | models.js |
```

Comment:
Optional: You could add more information about creating and deleting a game. For instance, you could add 

> {
>   title: 'California Games',
>   genre: 'Sports',
>   releaseDate: 'June 1987'
> }

to clarify the structure of the request.